Dockerfile

```diff
@@ -3,35 +3,23 @@
 #
 # http://github.com/aerospike/aerospike-tools.docker
 #
-FROM debian:bullseye-slim AS build
+FROM debian:bookworm-slim
 
 ARG TARGETARCH
 
 RUN \
   apt-get update -y \
   && apt-get install -y \
   wget \
-  build-essential \
-  zlib1g-dev \
-  libncurses5-dev \
-  libgdbm-dev \
-  libnss3-dev \
-  libssl-dev \
-  libreadline-dev \
-  libffi-dev \
-  libsqlite3-dev \
-  libbz2-dev
-
-# Build and Install Python from source
-RUN \
-  wget https://www.python.org/ftp/python/3.10.9/Python-3.10.9.tgz \
-  && tar -xf Python-*.tgz \
-  && cd Python-*/ \
-  && ./configure --enable-optimizations \
-  && make
+  python3 \
```

Comment:
What is the default python installed on Debian 12?

Reply:
on deb 12 slim there isn't one

Reply:
I mean, what version of python is installed when you install `python3`

Reply:
Python 3.9.16

Reply:
Hmmm, interesting. I ran `docker run --rm -ti --name deb12 debian:bookworm-slim bash` and got 3.11.2. Maybe they updated the base image since you last downloaded it?

Reply:
Rebuilding with --no-cache and --pull I still get 3.9.16. I get 3.11.2 if I run your command though. Not sure what's up. What version does asadm package? Is it possible that is shadowing what is installed?

Reply:
@dwelch-spike my bad, I was using the wrong bash command and getting the Python version of the host shell. It does show Python 3.11.2